pkg/ddc/juicefs/runtime_info_test.go

```diff
@@ -169,4 +177,4 @@ func TestJuiceFSEngine_getRuntimeInfo(t *testing.T) {
 			t.Errorf(" want %t, got %t", testCase.isErr, isErr)
 		}
 	}
-}
+}
```

Comment:
Please keep a return line after the method TestJuiceFSEngine_getRuntimeInfo as beflore.

Reply:
@waloc123 Please add a return line in the end of file: [pkg/ddc/juicefs/runtime_info_test.go]